.github/workflows/development-environment.yml

```diff
@@ -11,12 +11,17 @@ on:
 
 jobs:
   dev-environment:
-    name: python set up
+    name: set up
```

Comment:
This change and the one above helps to fit the full string name in the dropdown since it now contains the OS name.

Before:
<img width="386" alt="image" src="https://user-images.githubusercontent.com/44410/107695613-db8c0d80-6c7e-11eb-8da0-81e2174f86d6.png">

After:
<img width="378" alt="image" src="https://user-images.githubusercontent.com/44410/107695566-cc0cc480-6c7e-11eb-8eb3-f275a4663fcd.png">